### PR TITLE
Fix default arguments for flux_estimator._set_norm_parameter

### PR DESCRIPTION
**Pull request by luca-giunti**
**Description**
This pull request addresses https://github.com/gammapy/gammapy/issues/3694

### CODE REVIEW

**Comment by QRemy** [comment]
if we redefine everything (value, frozen, min, max interp,  scan_values, scan_min, scan_max) in this function then norm does not have to be in the arguments

File: gammapy/estimators/flux.py [hunk 1]
```diff
@@ -88,9 +88,15 @@ def _set_norm_parameter(self, norm=None, scaled_parameter=None):
...
-        norm.min = scaled_parameter.min / scaled_parameter.value
-        norm.max = scaled_parameter.max / scaled_parameter.value
-        norm.interp = scaled_parameter.interp
+        if scaled_parameter:
+            norm.min = scaled_parameter.min / scaled_parameter.value
+            norm.max = scaled_parameter.max / scaled_parameter.value
+            norm.interp = scaled_parameter.interp
+        else:
+            norm.min = self.norm_min
+            norm.max = self.norm_max
```
Comment (QRemy) [inline gammapy/estimators/flux.py:97]: the FluxEstimator defaults for norm_mina and norm_max are 0.2, and 5 respectively, so the constraint may be too strong if the reference model is not optimal then NaN default of the parameter may be better.

**Comment by adonath** [comment]
Conclusion from dev meeting: for we require spectral models to either define a `norm` or `amplitude` parameter. To make it independent from the parameter name we could introduce `Parameter(..., is_amplitude=True)` as well.

**Comment by adonath** [comment]
After #3860 one should modify https://github.com/gammapy/gammapy/blob/master/gammapy/estimators/flux.py#L116 to not check for the amplitude name, but for the `.is_norm` property instead. If it it not present it should fail... @luca-giunti Can you update the PR to do this?

**Comment by adonath** [comment]
Superseded by #3905.

**Pull request closed without merging**